src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -66,8 +66,10 @@ public ReadOnlyAddressBook getAddressBook() {
         return model.getAddressBook();
     }
 
+
     @Override
     public ObservableList<Student> getFilteredPersonList() {
+        System.out.println("logic manager: \n" + model.getFilteredPersonList());
```

Comment:
What is the impact of this line of code to the system? Is it mainly for debugging purposes or are there any other dependencies on this line of code?

Reply:
Nice catch. That's a line for debugging.